modules/web-mercator/docs/developer-guide/web-mercator-guide.md

```diff
@@ -1,6 +1,6 @@
 # Overview
 
-> The viewport-mercator-project module was added to math.gl on Oct 1, 2019. It previously resided in a separate repository. See the original [release history](https://github.com/uber-common/viewport-mercator-project/blob/master/docs/whats-new.md)
+> The `viewport-mercator-project` module was added to `@math.gl/web-mercator` on Oct 1, 2019. It previously resided in a separate repository. See the original [release history](https://github.com/uber-common/viewport-mercator-project/blob/master/docs/whats-new.md)
```

Comment:
Maybe clarify description somewhat? E.g: 

The `@math.gl/web-mercator` module is the replacement of the `viewport-mercator-project` repository, which is now deprecated. The contents of `viewport-mercator-project` was copied to this module on ..., and further development is taking place here.